### PR TITLE
WIP: Accessibility audit

### PR DESCRIPTION
**Pull request by helenb**

### CODE REVIEW

File: src/components/tag/tag.module.scss [hunk 1]
```diff
@@ -6,19 +6,19 @@
     border: 2px solid rgba(35, 23, 73, 0.05);
     border-radius: 5px;
     font-size: 12px;
-    color: rgba(35, 23, 73, 0.6);
+    color: #757575;
```
Comment (chris-lawton) [inline src/components/tag/tag.module.scss]: is there a variable we can use here?

File: src/images/man-fruit.svg [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<svg id="Layer_1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 541 551">
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 541 551">
```
Comment (chris-lawton) [inline src/images/man-fruit.svg]: I wonder if some of these should be given titles for accessibility

**Comment by chris-lawton** [comment]
couple of small comments but looking good overall

File: src/images/man-fruit.svg (hunk 2 above)
Comment (helenb) [reply src/images/man-fruit.svg]: Good idea - I've added it to my to dos for the accessibility work (I may get this branch merged first so I can sort out any merge conflicts with Nathan's changes first)

File: src/components/tag/tag.module.scss (hunk 1 above)
Comment (helenb) [reply src/components/tag/tag.module.scss]: Now added.

**Pull request merged**